src/modules/flow/main.c

```diff
@@ -108,10 +108,10 @@ volatile uint32_t boot_time10_us = 0;
 #define TIMER_LPOS		8
 #define MS_TIMER_COUNT		100 /* steps in 10 microseconds ticks */
 #define LED_TIMER_COUNT		500 /* steps in milliseconds ticks */
-#define SONAR_TIMER_COUNT 	100	/* steps in milliseconds ticks */
+#define SONAR_TIMER_COUNT	9999/* steps in milliseconds ticks */
```

Comment:
Whops, not intended to be included. I did this since we don't use this data at all, as the sonar is actually really bad.